lib/breakers/dashboard.rb

```diff
@@ -0,0 +1,53 @@
+require 'sinatra/base'
+require 'json'
+
+require 'byebug'
+
+module Breakers
+  class Dashboard < Sinatra::Base
```

Comment:
Maybe I'm misunderstanding, but I think a Rails engine would be a lot heavier. A sinatra app is mountable in the controller much like an engine would be. And I'm not sure how it could be a developer dependency only if it's going to be used by clients of the gem.

This dashboard thing may go away once the devops folks get prometheus and grafana set up, but for now I'm torn between pulling it out as a separate gem and just not worrying about it too much.